installer/manifests/keptn/charts/control-plane/templates/api-gateway-nginx.yaml

```diff
@@ -77,7 +77,7 @@ data:
       proxy_set_header X-Forwarded-Proto $scheme;
     }
 
-    location {{ .Values.prefixPath }}/bridge {
+    location ~* {{ .Values.prefixPath }}/bridge/.* {
```

Comment:
this would change the previous behaviour a little bit in that requests that end with `*bridge/` won't be proxied anymore since the `/` is required in the URL now. don't know if those requests are something we need or not though...

Reply:
thb, i don't know either. This change was brought up by @thisthat. What's your opinion on that?

Reply:
I would say that this is a nice side-effect bug fix 😉
This way, we are more strict and only URLs with `/bridge/` are redirected to it.